IFC4x3/Properties/d/DistanceFromRopeToFixedPulley_3CEvujvXv4h8b1bdTGctiY/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="DistanceFromRopeToFixedPulley_3CEvujvXv4h8b1bdTGctiY" Name="DistanceFromRopeToFixedPulley" UniqueId="cc3b9e2d-e61e-44ac-8941-9677509b7b22" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Extract from geometry?

Reply:
seconded...

Reply:
@fhulin

Reply:
Same comment as the other distance. We need the 2 property.. 08487af509e4319afa5d187d589ae5dfd92de558